maps/kleibkhar/kleibkhar_defines.dm

```diff
@@ -33,4 +33,4 @@
 	salary_modifier = 0.2
 
 /datum/map/kleibkhar/get_map_info()
-	return "You're a colonist on the planet Kleibkhar, a lush planet targeted for development by a corporation to create a new consumer base. No government has authority in this sector. Colonists come from a wide variety of backgrounds, but universally with only the shirt on their backs."
\ No newline at end of file
+	return "Kleibkhar Independant Colony. A diverse new commercial venture on the fringe."
```

Comment:
Independant -> Independent.

on the fringe -> on the fringe of known space.

Reply:
Fixed